src/app/components/BibPage/BibMainInfo.jsx

```diff
@@ -1,34 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router';
 import { isEmpty as _isEmpty } from 'underscore';
 
 const BibMainInfo = ({ bib = {} }) => {
   if (_isEmpty(bib)) return null;
 
-  const title = bib.title && bib.title.length ? bib.title[0] : '';
   const materialType = bib && bib.materialType && bib.materialType[0] ?
     bib.materialType[0].prefLabel : null;
+  const author = bib.creatorLiteral && bib.creatorLiteral.length ?
+    `${bib.creatorLiteral[0]}` : '';
+  const contributor = bib.contributorLiteral && bib.contributorLiteral.length ?
+    bib.contributorLiteral : null;
+  const yearPublished = bib && bib.dateStartYear ? bib.dateStartYear : null;
+  const placeOfPublication = bib && bib.placeOfPublication ? bib.placeOfPublication : null;
+  const publisher = bib && bib.publisher ? bib.publisher : null;
   const language = bib && bib.language && bib.language[0] ?
     bib.language[0].prefLabel : null;
-  const yearPublished = bib && bib.dateStartYear ? bib.dateStartYear : null;
-  const usageType = bib && bib.actionType && bib.actionType[0] ?
-    bib.actionType[0].prefLabel : null;
 
   return (
-    <div className="nypl-item-details">
-      <h1>{title}</h1>
-      <div className="nypl-item-info">
-        <p>
-          <span className="nypl-item-media">{materialType}</span>
-          {language && ` in ${language}`}
-        </p>
-        <p>{bib.extent} {bib.dimensions}</p>
-        <p>
-          {bib.placeOfPublication} {bib.publisher} {yearPublished}
-        </p>
-        <p className="nypl-item-use">{usageType}</p>
-      </div>
-    </div>
+    <span>
```

Comment:
Regarding your comment about how the top, above the fold, section was re-used before, I think it was and we can do it now. You can use the `DefinitionList` component and pass data in the form of 

```
[
  {
    term: 'FORMAT',
    definition: (
      <span>
         {materialType}
         {language && ` in ${language}`}
       </span>
    )
  },
  {
    term: 'AUTHOR',
    definition: (
      <Link
        onClick={e => this.newSearch(e, author)}
        title={`Make a new search for: ${author}`}
        to={`/search?${author}`}
      >
        {author}
      </Link>
    )
  },
  {
   // etc
  }
]
```

It's _similar_ to what the `BibDetails` component does when it makes the data structure. You can do the same so that you don't hardcode the structure - if that helps.